app/views/admin/reports/index.html.erb

```diff
@@ -9,6 +9,12 @@
   <li><%= link_to "Meter Report with deprecated meters included", admin_reports_amr_validated_readings_path(all_meters: true) %></li>
 </ul>
 
+<h2>Users</h2>
+
+<ul>
+  <li><%= link_to "Consents Granted", admin_consent_grants_path %></li>
```

Comment:
We already have this link from the admin "Admin" page - do we need it in both places..?

Reply:
(Maybe remove from other page..?)